addons/ness-lambda/cookiecutter.json

```diff
@@ -0,0 +1,27 @@
+{
+    "root": ".",
+    "lambda_name": "sample",
+    "lambda_underscore": "@{{ cookiecutter.lambda_name.lower().replace(' ', '_').replace('-', '_')}}",
```

Comment:
> I don't like a lot the name. What about `lambda_name_normalized` ?

Then it should be `lambda_name_terraform_normalized` because it's just normalized for terraform. For AWS resource name it's ok.
I could remove the " " replacement and make a check that the name contain no space/special characters

Reply:
👍